app/containers/HomeContainer/index.js

```diff
@@ -56,6 +63,8 @@ export function HomeContainer({
   useInjectReducer({ key: 'homeContainer', reducer });
   useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
+  const [text, setText] = useState('');
```

Comment:
unless you're doing some processing on this value the initial state should be null not an empty string